src/inmanta/ast/statements/__init__.py

```diff
@@ -208,9 +208,12 @@ def _add_to_dataflow_graph(self, graph: Optional[DataflowGraph]) -> None:
 
 
 class Literal(ExpressionStatement):
+    lexpos: Optional[int]
```

Comment:
Why not declare the type in the constructor? I believe this technically declares it as a class attribute as well though in practice it makes little difference of course.

Reply:
no idea. I fixed it.

Adding this is correct, no?

Reply:
Adding what is correct?

Reply:
lexpos :)

Reply:
Oh, for some reason I hadn't realized you'd added this, I thought you'd only added the typing. I'll double check.

Reply:
Yeah, looks like it is set by the parser. I find it a bit strange but it certainly corresponds with the actual current type.